src/main/java/org/broadinstitute/hellbender/utils/read/GoogleGenomicsReadToGATKReadAdapter.java

```diff
@@ -39,13 +39,25 @@
  */
 public final class GoogleGenomicsReadToGATKReadAdapter implements GATKRead {
 
-    private final Read genomicsRead;
-    private final UUID uuid;
+    private Read genomicsRead;
+    private UUID uuid;
 
     public GoogleGenomicsReadToGATKReadAdapter( final Read genomicsRead ) {
         this(genomicsRead, UUID.randomUUID());
     }
 
+    private GoogleGenomicsReadToGATKReadAdapter() {}
```

Comment:
Is this constructor ever used? If not, the instance variables do not need to be changed from final.